### PR TITLE
(feat) Speed Limit Override

### LINKED ISSUE #1957: [Request]: Bypass Max Speed
### Problem

_No response_

### Solution

Bypass vehicle speed caps

### Reason

Make cars go vr0000m more

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by T14D3**
Closes #1957

### CODE REVIEW

File: src/gui/handling/current_profile/current_profile_other.cpp [hunk 1]
```diff
@@ -23,8 +27,12 @@ namespace big
 				g_local_player->m_vehicle->m_handling_data->m_inertia_mult.y = fInertiaMult[1];
 				g_local_player->m_vehicle->m_handling_data->m_inertia_mult.z = fInertiaMult[2];
 			}
+			ImGui::Text("Speed Limit Override");
+			ImGui::SliderFloat("km/h", &speedLimitKMH, 0.f, 1000.f);
```
Comment (Yimura) [inline src/gui/handling/current_profile/current_profile_other.cpp:31]: People who use imperial won't understand this properly, also this feature should probably be added under the vehicle tab and not under handling.

Combined with that you don't save the speed override to the handling JSON file.
And you're not following coding guidelines with the capitalisation of your variables.

**Review by Yimura — CHANGES_REQUESTED** [review]

File: src/gui/handling/current_profile/current_profile_other.cpp (hunk 1 above)
Comment (T14D3) [reply src/gui/handling/current_profile/current_profile_other.cpp:31]: Right, should probably move the thing to the vehicle tab, will do
For the unit thing, I'll look into just pulling from whatever the user has set for the speedometer (km/h vs mp/h) and use that (and fix some capitalisation I guess)

Comment (binded-zz) [reply src/gui/handling/current_profile/current_profile_other.cpp:31]: does the speed work for flying stuff as well

Comment (ScottiePimp) [reply src/gui/handling/current_profile/current_profile_other.cpp:31]: > 

Simple fix for that. Add a bool check for which speed measurement the user has.

Then do:
```
float Speed = speedLimitInput / (imperial ? 2.236936f : 3.6f); // Convert to MPS
SET_ENTITY_MAX_SPEED(your_vehicle, Speed);
```

**Comment by Yimura** [comment]
PR has gone stale.

**Pull request closed without merging**